src/functions.php

```diff
@@ -133,7 +133,7 @@ function default_user_agent()
     static $defaultAgent = '';
 
     if (!$defaultAgent) {
-        $defaultAgent = 'GuzzleHttp/' . Client::VERSION;
+        $defaultAgent = 'GuzzleHttp/Guzzle';
```

Comment:
Maybe add `7.x` here?

Reply:
Then we still need to worry about the version. For what benefit?

Reply:
You could always do the trick php-cs-fixer does to detect its version?

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.16.1/src/ToolInfo.php#L61-L72

Reply:
That is pretty cool. But Im not sure we want the version at all. 

Anyhow. A followup PR may fix this.